modules/islandora_scholar_embargo/islandora_scholar_embargo.module

```diff
@@ -311,8 +311,8 @@ EOQ;
     }
     // XXX: This needs to be removed as it is no longer shipped with the module
     // and is considered deprecated of 7.x-1.6.
-    if (isset($params['dsid']) && isset($item[$dsid])) {
-      rules_invoke_event('islandora_scholar_datastream_embargo_expired', $item, $item[$dsid]);
+    if (isset($params['dsid'])) {
+      rules_invoke_event('islandora_scholar_datastream_embargo_expired', $item, $params['dsid']);
```

Comment:
Same deal, this is expecting an `islandora_datastream`, not what is probably a string?: https://github.com/Islandora/islandora_scholar/blob/ab835969c995be48bb5d3e44268bcb37c686a008/modules/islandora_scholar_embargo/islandora_scholar_embargo.rules.inc#L40